marketplace_builder/views/pages/how-platformos-works/platformos-workflow/marketplace-kit.liquid

```diff
@@ -0,0 +1,181 @@
+---
+converter: markdown
+metadata:
+  title: Marketplace Kit
+  description: 
+slug: how-platformos-works/platformos-workflow/marketplace-kit
+searchable: true
+---
+
+The **[Marketplace Kit](https://github.com/mdyd-dev/marketplace-kit)** is a command line interface that helps you deploy configuration files and assets to your platformOS site. 
+
+## Requirements 
+
+Marketplace Kit requires [Node.js](https://nodejs.org/en/) version 8 or higher to work properly.
+
+## Installation
+
+Follow our [Installing the Marketplace Kit](/get-started/installing-marketplace-kit) tutorial to install the Marketplace Kit. 
+
+## Commands and options
+
+All commands should be run in the project root directory (one level above the `marketplace_builder` directory). 
+
+**Usage**: 
+
+<pre class="command-line" data-user="user" data-host="host"><code class="language-bash">
+marketplace-kit [options] [command]
+</code></pre>
+
+**Help**: 
+
+The `help [cmd]` displays help for the command in `[cmd]`, for example:
+<pre class="command-line" data-user="user" data-host="host"><code class="language-bash">
+marketplace-kit help data
+</code></pre>
+
+### Options
+
+* `-V`, `--version`: Display the version number
+* `-h`, `--help`: Display usage information
+
+### Commands
+
+#### audit
+Check your code for deprecations, recommendations, errors. 
+
+| Command 	| Usage                 	| Example               	|
+|---------	|-----------------------	|-----------------------	|
+| audit   	| `marketplace-kit audit` 	| <pre class="command-line" data-user="user" data-host="host"><code class="language-bash">marketplace-kit audit</code></pre> 	|
+
+#### data
+Export, import or clean data on instance. 
+
+**Operations**: 
+
+* `export <environment>`: export Instance data to a JSON file
+* `import <environment>`: import Instance data from a JSON file
+* `clean <environment>`: remove all stored data in users, models, etc. Execute with caution, there is no coming back. 
```

Comment:
Apparently this will work only on staging, worth noting

Reply:
Thanks, added.